tests/oss-fuzz/pdu_parse_target.c

```diff
@@ -5,6 +5,7 @@ LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
     coap_pdu_t *pdu = coap_pdu_init(0, 0, 0, size);
     if (pdu) {
         coap_pdu_parse(COAP_PROTO_UDP, data, size, pdu);
+        coap_show_pdu(pdu);
```

Comment:
(Just a heads-up: PR #219 will add the log level as first paramenter to `coap_show_pdu()`).

Reply:
@bshastry Now that PR #219 has been merged, can you do a rebase and add the desired log-level as first parameter to `coap_show_pdu()`?

Reply:
Does this look good wrt #219 

```
int
LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
    coap_pdu_t *pdu = coap_pdu_init(0, 0, 0, size);
    if (pdu) {
        coap_pdu_parse(COAP_PROTO_UDP, data, size, pdu);
        coap_show_pdu(LOG_DEBUG, pdu);
        coap_delete_pdu(pdu);
    }
    return 0;
}
```